### ISSUE TITLE
Voice latency using HTC Desire

### ISSUE THREAD

**Issue by GoogleCodeExporter** [sonuarya/csipsimple#968]
```
What steps will reproduce the problem?
1. Place test call (echo test) to a local asterisk server
2. Observe latency between your voice and returned echo
3.

What is the expected output? What do you see instead?
Immediate echo as there's no network / server delay.
Echo returns only after +500ms

What version of the product are you using? On what operating system?
HTC Desire (stock rom froyo OR gingervillain) and CSipSimple trunk / ginger.

Please provide any additional information below.
- Identical with stock ROM (froyo) and gingervillain alt ROM
- Using same local echo test with native Gingerbread SIP, latency is same (even 
a little higher)
- Using same echo test with PAP2T + analog phone, there's just no latency
Might this be an audio driver issue with HTC firmware ?

```

Original issue reported on code.google.com by `damien.m...@gmail.com` on 18 May 2011 at 12:03
- Merged into: #954
